compiler/test/test_end_to_end.re

```diff
@@ -251,7 +251,7 @@ let basic_functionality_tests = [
   te("comp_bool2", "if (2 > true) {3} else {4}", "type"),
   te("comp_bool3", "if (true >= 4) {3} else {4}", "type"),
   te("comp_bool4", "let x = true; if (x < 4) {3} else {5}", "type"),
-  t("void", "{'foo';}", "void"),
+  t("void", "{\"foo\";}", "void"),
```

Comment:
Opinions on using `{| |}` instead of escaping all these double quotes?
```suggestion
  t("void", {|{"foo";}|}, "void"),
```

Might look a little weird wrapping the record/block syntax

Reply:
Made https://github.com/grain-lang/grain/issues/479 to address!